Hema/lib/first_project/power.rb

```diff
@@ -0,0 +1,15 @@
+module Hema
+  class Power
```

Comment:
We spoke in private before but I rather put it here as well.
We don't usually create a class if we don't expect it to be ever instantiated it and for that we usually just make a module instead. Like this:
```ruby
module Hema
  module Power
    def pow(...)
      ...
    end
  end
end
```
And from outside called like: `Hema::Power.pow(...)`